### ISSUE TITLE
Issue with people turning to bugs and or crashing 

### ISSUE THREAD

**Issue by kowalski713** [alexguirre/CScenarioPoint-Patch#6]

**Comment by alexguirre** [comment]
Any info on how to reproduce the bugs or crashes? Did it start with the latest GTAV update (1868.1)?

**Comment by kowalski713** [comment]
yeah it started with the new update and its when its used with world of variety IV pack Vanilla works and dispatch works not sure if it happens with just WOV or WOV and IV

**Comment by kowalski713** [comment]
honestly i couldnt reproduce it but i know some people on discord of WOV and on GTA 5 mods were saying it was happening sorry to waste your time

**Comment by kowalski713** [comment]
So dont sure if this is Cscenario 
But ive talked to someone on discord and the bought retail version and have Redux/Wov/Forest of san andress/gta5korn car pack they did all the stuff to get no crash at start up but the have a weird bug with peds
![Uploading image0.jpg…](sorry for the bad quality) im going to try and reproduce when i have time

**Comment by alexguirre** [comment]
@kowalski713 I can`t see the image, I guess hit the comment button too soon and didn't let it upload the image fully.

**Comment by kowalski713** [comment]
http://imgur.com/gallery/44Rjv1M here sorry about that and i missed a mod he is also using the Enchanced trainer

**Comment by alexguirre** [comment]
@kowalski713 That happens when a ped model spawns in an animal scenario (it tries to play an animal animation on an human model).
I only had that happen without this patch when I add more ped modelsets than what the vanilla game allows. But after installing the patch, it is fixed.
I will try to test those mods when I have time.

**Comment by darksmizar** [comment]
I'm experiencing crashes just after the loading screen ends, it's definitely CScenario, I've narrowed it down to that. I'm using the EGS version so it might have something to do with that. 
I'm using a new Gameconfig, Packfile adjuster and Heap  adjuster.

**Comment by darksmizar** [comment]
I'm experiencing crashes just after the loading screen ends, it's definitely CScenario, I've narrowed it down to that. I'm using the EGS version so it might have something to do with that.
I'm using a new Gameconfig, Packfile adjuster and Heap adjuster.
